### DISCUSSION TITLE
asp.net support?

### DISCUSSION THREAD

**Discussion by RageStone** [vercel/community#2320] [Help] [answered]
### Summary

Im looking for a web hosting that can provide support to .aspx files. Is vercel supports it?
And if anyone knows other web hosting services that support .aspx files, please let me know. Thanks!

**Comment by amyegan** [comment, marked as answer]
Hi @RageStone! We don't currently have a .NET runtime. You can find the current list of available runtimes on our [Serverless Function Runtimes](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#) page. If you're feeling ambitious, you could [develop your own runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#developing-your-own-runtime) and eventually share it as a [community runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#community-runtimes). 

I'm sure there are plenty of options that currently support .NET, but I don't know them well enough to be able to make a good recommendation. I imagine [Azure](https://azure.microsoft.com/en-us/) would be the easiest option for .NET compatibility since it's also in the Microsoft family 🙂
